core/sdk/src/api/device/panel/device_hdmi.rs

```diff
@@ -0,0 +1,42 @@
+use crate::api::device::device_request::DeviceRequest;
+use crate::api::firebolt::panel::fb_hdmi::HdmiSelectOperationRequest;
+use crate::extn::extn_client_message::{ExtnPayload, ExtnPayloadProvider, ExtnRequest};
+use crate::framework::ripple_contract::{ContractAdjective, RippleContract};
+use serde::{Deserialize, Serialize};
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub enum HdmiRequest {
+    GetAvailableInputs,
+    StartHdmiInput(HdmiSelectOperationRequest),
+    StopHdmiInput(HdmiSelectOperationRequest),
+}
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+#[serde(rename_all = "snake_case")]
+pub enum AVInputAdjective {
+    Hdmi,
+    Ota,
+    Composite,
+}
```

Comment:
Why is this defined in an HDMI module? When implementing Composite for example I would not expect to find the adjective for it defined here in HDMI.

Reply:
yeah AVInput probably needs a super .rs file for OTA and composite

Reply:
Moved AVInputAdjective to a super .rs file.